docs/get_vsphere_vm_details.md

```diff
@@ -0,0 +1,30 @@
+# get_vsphere_vm_details
+
+Retrieve details for a virtual machine.
+```py
+def get_vsphere_vm_details(self, id, timeout=15):
+```
+
+## Arguments
+| Name        | Type | Description                                                                 | Choices |
+|-------------|------|-----------------------------------------------------------------------------|---------|
+| id  | str  | ID of the virtual machine.    |         |
```

Comment:
We should use VM Name as a primary search option and then dynamically look up the ID.

Reply:
changed to vm_name, thanks